{"uuid":"ec9eaef7-e094-4975-91cc-c0218340886c","historyId":"b53946ce50fa9c2eafa03bb4900e6eab","fullName":"com.example.deleteUser.testDeleteUser","labels":[{"name":"package","value":"com.example.deleteUser"},{"name":"testClass","value":"com.example.deleteUser"},{"name":"testMethod","value":"testDeleteUser"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"GoRestApi"},{"name":"subSuite","value":"com.example.deleteUser"},{"name":"host","value":"SHYAMs-MacBook-Pro.local"},{"name":"thread","value":"6460@SHYAMs-MacBook-Pro.local.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"Delete multiple users from CSV"},{"name":"feature","value":"User Deletion"},{"name":"epic","value":"User Management API Tests"},{"name":"severity","value":"critical"}],"links":[],"name":"testDeleteUser","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"User deletion failed expected [204] but found [404]","trace":"java.lang.AssertionError: User deletion failed expected [204] but found [404]\n\tat org.testng.Assert.fail(Assert.java:111)\n\tat org.testng.Assert.failNotEquals(Assert.java:1578)\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:150)\n\tat org.testng.Assert.assertEquals(Assert.java:132)\n\tat org.testng.Assert.assertEquals(Assert.java:1419)\n\tat org.testng.Assert.assertEquals(Assert.java:1383)\n\tat com.example.deleteUser.testDeleteUser(deleteUser.java:30)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:739)\n\tat org.testng.TestRunner.run(TestRunner.java:614)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\n\tat org.testng.TestNG.run(TestNG.java:1079)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\n"},"stage":"finished","description":"Reads user IDs from CSV and deletes them via API","steps":[],"attachments":[{"name":"Request","source":"0257f7e8-3689-4577-bce5-1928bcd054d7-attachment.html","type":"text/html"},{"name":"HTTP/1.1 404 Not Found","source":"80d1104d-5dc7-4d13-b766-a51d25c8b712-attachment.html","type":"text/html"},{"name":"Response Status Code","source":"df418ed4-d806-49bc-ab1a-dec72b67a91b-attachment.txt","type":"text/plain"}],"parameters":[],"start":1741534213660,"stop":1741534217602}